app/assets/javascripts/angular/app.js

```diff
@@ -1,5 +1,40 @@
 'use strict';
 
 angular.module('mentorhub', [
-    'mentorhub.board'
-]);
\ No newline at end of file
+        'mentorhub.board'
+    ])
+
+    .filter('capitalize', function() {
+        return function(input) {
+            return (!!input) ? input.charAt(0).toUpperCase() + input.substr(1).toLowerCase() : '';
+        }
+    })
+
+    .filter('date_filter', function () {
+        return function (input) {
+            return new Date(input)
+        }
+    })
+
+    .factory('ApiUrls', function () {
+        var apiBaseUrl = '/';
+        var baseUrls = {
+            'update_section': apiBaseUrl + 'tracks/{track_id}/section_interactions/{section_id}',
+            'create_todo': apiBaseUrl + 'tracks/{track_id}/section_interactions/{section_id}/todos',
+            'update_todo': apiBaseUrl + 'tracks/{track_id}/section_interactions/{section_id}/todos/{todo_id}'
+        };
+
+        return angular.extend(baseUrls, {});
```

Comment:
@Vishal0203 Why do we need this? `baseUrls` is a regular JS object already. So, why are we extending it with an empty object?